packages/mapviewer/src/modules/drawing/components/DrawingToolbox.vue

```diff
@@ -310,9 +310,17 @@ const debounceSaveDrawingName = debounce(async (newName) => {
             v-if="showClearConfirmationModal"
             show-confirmation-buttons
             fluid
+            confirm-key="confirm_delete"
+            confirm-icon="far fa-trash-alt"
             @close="onCloseClearConfirmation"
         >
             {{ t('confirm_remove_all_features') }}
+            <br />
```

Comment:
please use a `<div>` instead of a `<br/>`

could you also wrap this warning message in an [alert block](https://getbootstrap.com/docs/5.3/components/alerts/)? That should give it more breathing room, and help users catch it